documentation/asciidoc/computers/camera/rpicam_options_common.adoc

```diff
@@ -1,29 +1,33 @@
-=== Common command line options
+=== Common options
 
 The following options apply across all the `rpicam-apps` with similar or identical semantics, unless otherwise noted.
 
-----
-	--help,		-h		Print help information for the application
-----
+To pass one of the following options to an application, prefix the option name with `--`. If the option requires a value, pass the value immediately after the option name, separated by a single space. If the value contains a space, surround the value in quotes.
 
-The `--help` option causes every application to print its full set of command line options with a brief synopsis of each, and then quit.
+Some options have shorthand aliases, for example `-h` instead of `--help`. Use these shorthand aliases instead of the full option name to save space and time at the expense of readability.
 
-----
-	--version			Print out a software version number
-----
+==== `help`
```

Comment:
I can see what you're trying to do here, by mentioning only-once about prefixing the options with double-dashes... unfortunately people are lazy and they don't read the _entire_ documentation from top to bottom, they just search / skim to find what they're looking for, and then copy-paste what they find. So some (_maybe_ not all?) people seeing this new version of the documentation would either try to use `rpicam-hello help` (and be confused why it doesn't work) or they would assume that they can _only_ use `rpicam-hello -h` (and not realise that `--help` exists).
IMVHO of course :slightly_smiling_face:

Reply:
I'll give this a real good thinkin'. While true, folks can always rely on the help output if they get stuck. Maybe I should toss a reminder at the top of each section to make it a little easier.

Reply:
Also, you seem to have removed a lot of the example command-usages. Never underestimate the value of working examples that users can just copy'n'paste?